### DISCUSSION TITLE
〔 Comments Section 〕Fallout Shelter Online: Attachments and Remaking

### DISCUSSION THREAD

**Discussion by morgosus** [morgosus/morgosus.github.io#14] [Comments]
A comments section for the [Fallout Shelter Online: Attachments and Remaking](https://www.toms.click/gaming/foso/2021/08/13/fallout-shelter-attachments.html) post.

**Comment by GHex06** [comment]
A more comprehensive guide in crafting attachments and it's connection to the Mystery treasure will greatly improve the post :)

**Comment by morgosus** [comment]
> A more comprehensive guide in crafting attachments and it's connection to the Mystery treasure will greatly improve the post :)

The next one will most likely be all about the Mystery Treasure.

**Reply by morgosus** [reply]
Done and done.

**Comment by Emptycz** [comment]
How does attachment swap work?

**Reply by morgosus** [reply]
It's simple. You grab two pieces of the same item (rarity and name), their stats may be different. Then, you "swap" their base stats. That means that the target item, which may be level 1, can be upgraded up to the same level (including rarity) that the used item is, basically for free, only using the reformatting chip.
